scripts/scr_initialize_custom/scr_initialize_custom.gml

```diff
@@ -1,1352 +1,1338 @@
-	    if (obj_creation.strength<=4) then ninth=0;
-	    if (obj_creation.strength<=3) then eighth=0;
-	    if (obj_creation.strength<=2) then seventh=0;
-	    if (obj_creation.strength<=1) then sixth=0;
-
-	    var bonus_marines,o;bonus_marines=0;o=0;
-	    if (obj_creation.strength>5) then bonus_marines=(obj_creation.strength-5)*50;
-
-	    /*repeat(20){
-	        if (bonus_marines>=5) and (veteran>0){bonus_marines-=5;veteran+=5;}
-	        if (bonus_marines>=5) and (second>0){bonus_marines-=5;second+=5;}
-	        if (bonus_marines>=5) and (third>0){bonus_marines-=5;second+=5;}
-	        if (bonus_marines>=5) and (fourth>0){bonus_marines-=5;second+=5;}
-	        if (bonus_marines>=5) and (fifth>0){bonus_marines-=5;second+=5;}
-	        if (bonus_marines>=5) and (sixth>0){bonus_marines-=5;second+=5;}
-	        if (bonus_marines>=5) and (seventh>0){bonus_marines-=5;second+=5;}
-	        if (bonus_marines>=5) and (eighth>0){bonus_marines-=5;second+=5;}
-	        if (bonus_marines>=5) and (ninth>0){bonus_marines-=5;second+=5;}
-	        if (bonus_marines>=5) and (tenth>0){bonus_marines-=5;second+=5;}
-	    }*/
-	}
-
-	if (obj_creation.custom!=0){
-	    var bonus_marines,o;bonus_marines=0;o=0;
-	    if (obj_creation.strength>5) then bonus_marines=(obj_creation.strength-5)*50;
-
-	    repeat(20){
-	        if (bonus_marines>=5) and (veteran>0){bonus_marines-=5;veteran+=5;}
-	        if (bonus_marines>=5) and (second>0){bonus_marines-=5;second+=5;}
-	        if (bonus_marines>=5) and (third>0){bonus_marines-=5;third+=5;}
-	        if (bonus_marines>=5) and (fourth>0){bonus_marines-=5;fourth+=5;}
-	        if (bonus_marines>=5) and (fifth>0){bonus_marines-=5;fifth+=5;}
-	        if (bonus_marines>=5) and (sixth>0){bonus_marines-=5;sixth+=5;}
-	        if (bonus_marines>=5) and (seventh>0){bonus_marines-=5;seventh+=5;}
-	        if (bonus_marines>=5) and (eighth>0){bonus_marines-=5;eighth+=5;}
-	        if (bonus_marines>=5) and (ninth>0){bonus_marines-=5;ninth+=5;}
-	        if (bonus_marines>=5) and (tenth>0){bonus_marines-=5;tenth+=5;}
-	    }
-	}
-
-
-
-
-	if (global.chapter_name="Salamanders"){veteran+=20;second+=20;third+=20;fourth+=20;fifth+=20;sixth+=20;seventh=0;eighth=0;ninth=0;tenth-=40;}
-	// if (global.chapter_name="Salamanders"){veteran+=12;second+=2;third+=2;fourth+=2;fifth+=4;sixth+=4;seventh+=4;eighth=0;ninth=0;}
-	if (global.chapter_name="Blood Angels"){chaap=3;apothecary=10;epistolary=6;codiciery=6;lexicanum=8;}
-	if (global.chapter_name="Dark Angels"){veteran=0;terminator+=80;}
-	if (global.chapter_name="Lamenters"){
-	    tenth=0;ninth=0;eighth=0;seventh=0;sixth=0;fifth=0;
-	    techs=8;epistolary=6;apothecary=5;codiciery=3;lexicanum=5;terminator=5;veteran+=10;
-	}
-	if (global.chapter_name="Soul Drinkers"){
-		tenth-=38;seventh=0;sixth=40; assault-=10;
-		fifth-=20;fourth-=20;third-=20;second-=20;terminator-=5;veteran-=20;
-	}
-if (global.chapter_name="Crimson Fists"){
```

Comment:
same as above

Reply:
same as above.fixed

Reply:
this is still deleted

Reply:
oh shit yeah sorry i made a comment here then had to run out into the yard before i physically made the push and was then tied up all day